README.md

```diff
@@ -63,6 +63,7 @@ claims no expertise beyond English.
 | ru     | Russian             | 4.0.0   | [DigitalWolf98][ru]   | Vladislav Rykhtikov
 | si     | Sinhala             | 4.0.3   | [npwsamarasinghe][si] | Nalaka Prasad
 | sk     | Slovak              | 4.0.0   | [xbotkaj][sk]         | Jozef Botka
+| sr     | Serbian             | 4.2.0   | [nebojsajsimic][sr]   | Nebojša Simić
```

Comment:
@paulbalandan The version is 4.3.3? 

@nebojsajsimic 
`[nebojsajsimic][sr]` does not work because you did not define it.
Please add something like this below the table.
```
[sr]: https://github.com/nebojsajsimic
```